restful.go

```diff
@@ -66,7 +66,10 @@ func (c *APIClient) NewDefaultCopyOptions() map[string]string {
 }
 
 type APIClient struct {
-	cli *http.Client
+	SessionID string
+	QuerySeq  int64
+	cli       *http.Client
+	rows      *nextRows
```

Comment:
I think we have no need to handle the  `rows.close`, just for user.

Reply:
we have to clean up the server with `final_url` and update session conf  according its resp.